src/components/elements/accordion/types.ts

```diff
@@ -21,6 +21,7 @@ export interface IAccordionSchema<V = undefined, C = undefined>
 	expanded?: boolean | undefined;
 	displayState?: BoxContainerDisplayState | undefined;
 	title: string;
```

Comment:
should `title` be set to optional now that it can be replaced with `titleSchema`?

Reply:
hmm how about we update title to support schema as well, this way there wont be overlapping behaviour

Reply:
combined